### DISCUSSION TITLE
import cannot be resolved for React

### DISCUSSION THREAD

**Discussion by Emokores256** [tailwindlabs/heroicons#796] [Help]
After running `npm i @heroicons/react`, I couldn't get the icons to display when I tried to use `import { BellIcon } from "@heroicons/react/24/outline"`. When I check the repo I cannot see the directory "24". I don't know if it's me missing something or if something is actually missing. I need help with this.

**Comment by BrunoEleodoro** [comment]
could you solve this?

**Comment by PtradeLLC** [comment]
Having issue with this as well. Any resolution?

**Comment by vitalijalbu** [comment]
me too on solid icons
